### ISSUE TITLE
Examples for append and flip functions

### ISSUE THREAD

**Issue by s0rbus** [chrisdone/jl#10]
I am a newbie to Haskell, but have found jl pretty straightforward to use and very useful. However I am struggling with 'append' and 'flip'. No matter what I try I get parsing or Inferer errors. Could you please provide simple usage examples of each? I have managed to use the 'equivalents' in the ghci repl using '<>' and 'flip (/)' so I think I understand what is supposed to happen, just can't get the syntax right in jl

**Comment by chrisdone** [comment]
Woops, I just noticed the type of `append` is wrong. Let me fix it!

**Comment by chrisdone** [comment]
Fixed in a2b23a0.

```haskell
-bash-3.2$ echo '"abc"' | jl '\x -> append x "y"'
"abcy"
-bash-3.2$ echo '"abc"' | jl '\x -> flip append x "y"'
"yabc"
```
